1_BayesIntuition.ipynb

```diff
@@ -0,0 +1,893 @@
+{
```

Comment:
<ul><li>but after "conditioning" on smoking was 0.1</li></ul><p> </p><ul><li>getting ~from~ cancer</li><li>Additionally what if we know something else <strong>about</strong> the patient,</li><li>...like gender (shouldn't be sex instead of gender?)</li><li>you say "In practice these numbers are typically unavailable, It's practically impossible to know the base rate of cancer of the world." but actually we have estimations for these numbers. Not sure what are you trying to say here, maybe that we have numbers plus uncertainty?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/canyon289/pyladies-bayes/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Thanks for first three correction. With sex vs gender I get them mixed up, I'll make change but can't say which one is proper</p><p> </p><p>For last point, if I were to get cancer base rate estimate where do you get this from? Doesn't uncertainty in base rate affect the final posterior point probability?</p><p> </p>

Reply:
probably from the WHO, right uncertainty in base rate will affect the final posterior, unless you choose to represent it using a number instead of a distribution. Is similar to assume your observed variable was measured without error

Reply:
Thanks!